.github/README_es.md

```diff
@@ -313,7 +313,7 @@ func main() {
 
 ```
 
-### Middleware logger
+### Middleware para Mantener Bitacoras
```

Comment:
The previous translation sounds better than any other "logger" spanish translation. And if you still wanting a spanish translation, "para Mantener Bitacoras" is not correctly translated.

Reply:
I'm from Mexico, and we actually just use several terms just in Spanish when there's not a clear translation (i.e. _logger, loggear, loggeo_, etc). However, I do have Spanish coworkers which use _bitacora_ instead.

I do agree that logger (or _registro_) sounds good. So I reverted this and changed it to _registro_ later on :v: